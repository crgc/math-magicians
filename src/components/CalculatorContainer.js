import Calculator from './Calculator';

const CalculatorContainer = () => (
  <div className="CalculatorContainer">
    <div className="math-quote">
      <h1>Let&apos;s do some math!</h1>
    </div>
    <Calculator />
  </div>
);

export default CalculatorContainer;
