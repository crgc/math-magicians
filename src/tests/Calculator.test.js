import React from 'react';
import renderer from 'react-test-renderer';
import CalculatorContainer from '../components/CalculatorContainer'

it('Calculator renders correctly', () => {
  const tree = renderer
    .create(<CalculatorContainer />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});