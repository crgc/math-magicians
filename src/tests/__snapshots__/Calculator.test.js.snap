// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator renders correctly 1`] = `
<div
  className="CalculatorContainer"
>
  <div
    className="math-quote"
  >
    <h1>
      Let's do some math!
    </h1>
  </div>
  <div
    className="Calculator"
  >
    <div
      className="Display"
    >
      0
    </div>
    <div
      className="Keyboard"
    >
      <div
        className="Key"
        onClick={[Function]}
        role="button"
      >
        AC
      </div>
      <div
        className="Key"
        onClick={[Function]}
        role="button"
      >
        +/-
      </div>
      <div
        className="Key"
        onClick={[Function]}
        role="button"
      >
        %
      </div>
      <div
        className="Key bg-orange"
        onClick={[Function]}
        role="button"
      >
        รท
      </div>
      <div
        className="Key"
        onClick={[Function]}
        role="button"
      >
        7
      </div>
      <div
        className="Key"
        onClick={[Function]}
        role="button"
      >
        8
      </div>
      <div
        className="Key"
        onClick={[Function]}
        role="button"
      >
        9
      </div>
      <div
        className="Key bg-orange"
        onClick={[Function]}
        role="button"
      >
        x
      </div>
      <div
        className="Key"
        onClick={[Function]}
        role="button"
      >
        4
      </div>
      <div
        className="Key"
        onClick={[Function]}
        role="button"
      >
        5
      </div>
      <div
        className="Key"
        onClick={[Function]}
        role="button"
      >
        6
      </div>
      <div
        className="Key bg-orange"
        onClick={[Function]}
        role="button"
      >
        -
      </div>
      <div
        className="Key"
        onClick={[Function]}
        role="button"
      >
        1
      </div>
      <div
        className="Key"
        onClick={[Function]}
        role="button"
      >
        2
      </div>
      <div
        className="Key"
        onClick={[Function]}
        role="button"
      >
        3
      </div>
      <div
        className="Key bg-orange"
        onClick={[Function]}
        role="button"
      >
        +
      </div>
      <div
        className="Key big-zero"
        onClick={[Function]}
        role="button"
      >
        0
      </div>
      <div
        className="Key"
        onClick={[Function]}
        role="button"
      >
        .
      </div>
      <div
        className="Key bg-orange"
        onClick={[Function]}
        role="button"
      >
        =
      </div>
    </div>
  </div>
</div>
`;
